src/providers/FriendProvider.tsx

```diff
@@ -100,10 +123,11 @@ function FriendProvider(props: Props): React.ReactElement {
   const [friendState, dispatch] = useReducer<Reducer>(reducer, initialState);
 
   const actions = {
+    // legacy
```

Comment:
Feel free to remove the provider if you think is no more needed!

Reply:
resolved [101e75c](https://github.com/dooboolab/hackatalk-mobile/pull/115/commits/101e75cde575e12e7f417b4ed32c0a519dc9eede)